.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop.yml
+- https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_base.yml
```

Comment:
Yes @durandom but I think we should be using the local file: `.rubocop_base.yml`, not the URL